data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+			`	He steps back and looks at the group. "May the Embers burn bright for you." With that, he gives the group a respectful nod and walks away. The captains all look around, make various signs you think are probably encouraging combat pilot slang, and variations on Prefect Chilia's parting words. As they disperse to their ships, one gestures to you. "Lead on, Captain."`
+				accept
+	npc kill
+		government "Korath"
+		personality staying
+		system "Nenia"
+		fleet
+			names "korath"
+			variant
+				"Korath World-Ship B (Ember)"
+					cargo "void sprite" 10 40
+				"Korath Raider (Ember)" 4
+					cargo "void sprite" 10 40
+		dialog "You have destroyed all the Korath that appeared to be hunting the Void Sprites."
+	npc
+		government "Remnant"
+		personality escort launching appeasing unconstrained disables plunders
+		fleet
+			names "remnant"
+			variant
+				"Albatross (Shield)"
+				"Albatross"
+				"Ibis" 2
+				"Starling (Heavy)" 6
+				"Petrel" 4
+				"Tern" 8
+	on enter "Aescolanus"
+		dialog `The Korath fleet is gone. Given the urgency Prefect Chilia put on this mission, the onus is on you to find it... Quickly.`
+	on complete
+		conversation
+			`The taskforce streams back into Caelian without fanfare. The ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they just did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`
+
+
+
+mission "Remnant: Cognizance 5"
+	name "Transport Plume to a meeting"
+	description "Prefect Chilia needs Plume to consult about the void sprites."
+	deadline 5
+	source "Caelian"
+	stopover "Aventine"
+	to offer
+		has "Remnant: Cognizance 4: done"
+	on offer
+		conversation
+			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`
```

Comment:
```suggestion
			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair, Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`
```